src/color/p5.Color.js

```diff
@@ -290,6 +294,34 @@ class Color {
     }
     return colorString;
   }
+/**
+   * Checks if two colors contrast ratio is WCAG 2.1 compliant and returns the ratio
+   *
+   * @param {Color} other
+   * @returns {{ ratio: Number, passes: boolean }}
+   * @example
+   * <div>
+   * <code>
+   * 
+   * function setup() {
+   *   // Define colors
+   *   let color1 = color(255, 255, 255);
+   *   let color2 = color(0);
+   *  
+   *   // Test for contrast
+   *   let result = color1.contrast(color2)
+   * 
+   *   console.log(result)
```

Comment:
On the p5.js reference, this will require users to open the console to see the result. Do you think we could show the result on the screen with something like `text()`?